cli/cmd/internal/client/apps.go

```diff
@@ -1 +1,47 @@
 package client
+
+import (
+	"code.cloudfoundry.org/quarks-utils/pkg/cmd"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/suse/carrier/cli/paas"
+)
+
+var ()
+
+// CmdApps implements the carrier app command
+var CmdApps = &cobra.Command{
+	Use:   "apps",
+	Short: "Lists all apps",
+	Args:  cobra.ExactArgs(0),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		app, cleanup, err := paas.BuildApp(cmd.Flags(), nil)
```

Comment:
[Minor] The use of "App" in this method's name is a bit confusing (initially I though it was a user's app). We could rename it to `BuildCarrier` or something like that. Could be another PR.

Reply:
Maybe `NewCarrierClient` ? Because that is the main result in the `app` variable, a `CarrierClient` structure.

Reply:
:+1: